env-setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+REPOBASE=$(realpath $(dirname "$0"))
+LIRIDIR="REPOBASE/.build"
+
+if [ ! -d "$LIRIDIR" ]; then
+  mkdir -p "$LIRIDIR"
+fi
+
+export LD_LIBRARY_PATH=$LIRIDIR/lib:$LD_LIBRARY_PATH
+export XDG_DATA_DIRS=$LIRIDIR/share:/usr/local/share:/usr/share:~/.local/share:~/.local/share/flatpak/exports/share
+export XDG_CONFIG_DIRS=$LIRIDIR/etc/xdg:/etc/xdg
+export QT_PLUGIN_PATH=$LIRIDIR/lib/plugins
+export QML2_IMPORT_PATH=$LIRIDIR/lib/qml:$QML2_IMPORT_PATH
+export PKG_CONFIG_PATH=$LIRIDIR/lib/pkgconfig:$PKG_CONFIG_PATH
+export PATH=$LIRIDIR/bin:$PATH
+
+if [ "$1" == "nvidia" ]; then
```

Comment:
`!=` not `==`

Reply:
Sorry,.... I didn't read the not in the comment :)

For people who ain't got an nvidia card, I guess this setting doesn't make a difference?